command/repo.go

```diff
@@ -361,6 +356,15 @@ func repoFork(cmd *cobra.Command, args []string) error {
 				return fmt.Errorf("did not understand argument: %w", err)
 			}
 
+		} else if strings.HasPrefix(repoArg, "git@") {
+			parsedURL, err := git.ParseURL(repoArg)
+			if err != nil {
+				return fmt.Errorf("did not understand argument: %w", err)
+			}
+			toFork, err = ghrepo.FromURL(parsedURL)
```

Comment:
You didn't introduce the `toFork` var in this PR. But naming it to `repoToFork` might make this easier to read. I was thrown off by what `toFork` did.